fastlane/lib/fastlane/actions/apteligent.rb

```diff
@@ -0,0 +1,90 @@
+module Fastlane
+  module Actions
+    class ApteligentAction < Action
+      def self.run(params)
+        command = []
+        command << "curl"
+        command += upload_options(params)
+        command << upload_url(params[:app_id])
+
+        # Fastlane::Actions.sh has buffering issues, no progress bar is shown in real time
+        # will reanable it when it is fixed
+        # result = Fastlane::Actions.sh(command.join(' '), log: false)
+        shell_command = command.join(' ')
+        result = Helper.is_test? ? shell_command : `#{shell_command}`
+        fail_on_error(result)
+        result
+      end
+
+      def self.fail_on_error(result)
+        if result.include?("error")
+          raise "Server error, failed to upload the dSYM file".red
```

Comment:
The server return html now with the same message "Server error, failed to upload the dSYM file" :cry: .